cmp/grid/columns/Column.ts

```diff
@@ -939,7 +939,7 @@ export class Column {
                         recordB,
                         column: this,
                         gridModel,
-                        defaultComparator: (a, b) => this.defaultComparator(a, b),
+                        defaultComparator: this.defaultComparator,
```

Comment:
If we are changing this from a fat arrow function, I think we need to do `this.defaultComparator.bind(this)`, so that the scope is correct for the implementation there.

Reply:
defaultComparator is already a fat arrow function